### DISCUSSION TITLE
ANSI color map in VSCode doesn't work directly, but work well when app opened in tmux.

### DISCUSSION THREAD

**Discussion by huiyeruzhou** [Textualize/textual#4336] [Q&A]
I think textual and vscode color map have a strange way to interative.

When I open my textual app in tmux, it looks like this:
![image](https://github.com/Textualize/textual/assets/92617158/ada32536-f02f-4efa-bfb3-c50ce3f92897)
When I directly open it in vscode terminal, it doesn't mapped by my vscode colormap...
![image](https://github.com/Textualize/textual/assets/92617158/26bd97a7-0c4a-4394-bbee-55415b5cac01)

A guess: tmux will show color using ANSI color, which is mapped by vscode configure. Textual, on the other hand, will show color using true-color.

**Comment by huiyeruzhou** [comment]
I wonder if I have got the key point to this problem, please share your idea! Any other trick about mapping colors in textual app(override button css can be very complex) will also help. Thanks for any help!

**Reply by davep** [reply]
It's not really clear to me what your actual question is here, as you seem to have identified the source of the difference.

But if you're asking: how do you control the colours of your application then, yes, you'd use CSS. If there's anything about [Textual's CSS](https://textual.textualize.io/guide/CSS/) or its [design system](https://textual.textualize.io/guide/design/) that isn't clear, do please ask for clarification.

**Reply by huiyeruzhou** [reply]
Hi! Thank you for your comment. My main question here is whether there's a method to customize ANSI color mapping similar to what was done in pull request #4192.
